enclave-tls/samples/enclave-tls-client/client.c

```diff
@@ -49,6 +53,78 @@ static sgx_enclave_id_t load_enclave(bool debug_enclave)
 #endif
 // clang-format on
 
+static uint8_t user_mrsigner[32];
+static uint8_t user_mrenclave[32];
+
+static int parse_hex(const char *hex, void *buffer, size_t buffer_size)
+{
+	if (!hex || !buffer || buffer_size == 0)
+		return -1;
+
+	if (strlen(hex) != buffer_size * 2) {
+		ETLS_ERR("Invalid hex string (%s) length\n", hex);
+		return -1;
+	}
+
+	for (size_t i = 0; i < buffer_size; i++) {
+		if (!isxdigit(hex[i * 2]) || !isxdigit(hex[i * 2 + 1])) {
+			ETLS_ERR("Invalid hex string '%s'\n", hex);
+			return -1;
+		}
+
+		sscanf(hex + i * 2, "%02hhx", &((uint8_t *)buffer)[i]);
+	}
+
+	return 0;
+}
+
+static int parse_config_file(char *path)
+{
+	FILE *fp;
+	char errbuf[200];
+
+	fp = fopen(path, "r");
+	if (!fp) {
+		ETLS_ERR("failed to open /opt/enclave-tls/config.toml\n");
```

Comment:
If the global config is not provided, the error exit leads to the program failure. I think this behavior should be configurable.